nvm.sh

```diff
@@ -2017,23 +2010,17 @@ nvm_die_on_prefix() {
   if [ -z "$NVM_COMMAND" ]; then
     nvm_err 'Second argument "nvm command" must be nonempty'
     return 2
-  fi
-
-  if [ -n "${PREFIX-}" ] && ! (nvm_tree_contains_path "$NVM_DIR" "$PREFIX" >/dev/null 2>&1); then
+  elif [ -n "${PREFIX-}" ] && ! (nvm_tree_contains_path "$NVM_DIR" "$PREFIX" >/dev/null 2>&1); then
     nvm deactivate >/dev/null 2>&1
     nvm_err "nvm is not compatible with the \"PREFIX\" environment variable: currently set to \"$PREFIX\""
     nvm_err 'Run `unset PREFIX` to unset it.'
     return 3
-  fi
-
-  if [ -n "${NPM_CONFIG_PREFIX-}" ] && ! (nvm_tree_contains_path "$NVM_DIR" "$NPM_CONFIG_PREFIX" >/dev/null 2>&1); then
+  elif [ -n "${NPM_CONFIG_PREFIX-}" ] && ! (nvm_tree_contains_path "$NVM_DIR" "$NPM_CONFIG_PREFIX" >/dev/null 2>&1); then
```

Comment:
I don't think we need all these elifs; I actually prefer the separate `if` blocks here.